pages/components/joinUsPage/buildYourCareer/BuildYourCareer.tsx

```diff
@@ -11,6 +11,14 @@ import { mainWrapper, imageWrapper, imageWrapperMobile,
 export const BuildYourCareer = () => {
   const data = useSelector((state: any) => state.joinUs.pageTexts);
 
+  const handleClick = () => {
+    const elem = document.getElementById("apply-for-job");
+    if(elem){
+      const height = elem.getBoundingClientRect().top + document.documentElement.scrollTop - 80;
```

Comment:
```suggestion
      const height = elem.getBoundingClientRect().top + document.documentElement.scrollTop - DYNAMIC;
```

80 will also gets broken if header will change its sizes